### ISSUE TITLE
Bug: Should not show Follow button in Personal Editions

### ISSUE THREAD

**Issue by chenilim** [mattermost/focalboard#2248]
## Steps to reproduce the behavior

1. Open a card in Personal Desktop or Personal Server
2. There's a "Follow" button, but clicking on it does nothing

## Expected behavior

Don't show a non-functioning Follow button.

## Edition and Platform

 - Edition: Personal Desktop / Personal Server
 - Version: v0.14 RC1
 - Browser and OS: Mac and Windows

## Additional context

Propose hiding the Follow button on Personal Editions.

**Comment by chenilim** [comment]
cc @wuwinson, @asaadmahmood, @wiggin77, @ogi-m - I think we discussed this briefly, but don't remember if there was a bug filed. While I don't think this is a top priority, we should fix this if it's easy to head-off any user confusion and bug reports.

**Comment by wiggin77** [comment]
@chenilim, just want to confirm that we'll only show the button for plugin, correct?

**Comment by chenilim** [comment]
Yup @wiggin77: We should only show the Follow button for the plugin (and not Personal Desktop or Personal Server, where the button doesn't work). Thanks.

I don't believe this affects Notifications, so it should be a matter of adding a condition to displaying the button.
